make-util.js

```diff
@@ -1442,7 +1442,13 @@ var createPushCmd = function (taskPublishFolder, fullTaskName, taskVersion) {
     var taskFeedUrl = process.env.AGGREGATE_TASKS_FEED_URL;
     var apiKey = 'Skyrise';
 
-    fs.writeFileSync(taskPushCmdPath, `nuget.exe push ${nupkgName} -source "${taskFeedUrl}" -apikey ${apiKey}`);
+    var pushCmd = `nuget.exe push ${nupkgName} -source "${taskFeedUrl}" -apikey ${apiKey}`;
+
+    if (process.env['courtesyPush']) {
```

Comment:
Does this just make it so that it doesn't error if the package exists? If so, we can just always add this regardless of whether or not it's `courtestyPush` mode.

Reply:
I thought about that, this option will error on lower versions of nuget though - would rather not introduce that dependency

Reply:
Also, note that it works without this option, this just makes sure the exit code doesn't get changed (so we don't fail the task)